src/CabanaPD_Particles.hpp

```diff
@@ -317,6 +325,12 @@ class Particles<MemorySpace, PMB, TemperatureIndependent, BaseOutput, Dimension>
         _timer.stop();
     }
 
+    auto numFrozen() const { return n_frozen; }
```

Comment:
These are meant to be offsets, correct? What is the presumed ordering here?

Reply:
Added comments